contracts/BridgeCommittee.sol

```diff
@@ -96,29 +116,36 @@ contract BridgeCommittee {
         } else if (messageType == BRIDGE_OWNERSHIP) {
             address newOwner = decodeBridgeOwnershipMessage(message);
             _transferBridgeOwnership(newOwner);
+        // Need to check the message type is legit (e.g. can't be 10000)
         } else {
             // if the message type is not for the committee, submit it to the bridge
             bridge.submitMessage(message);
         }
-        emit MessageProcessed(message);
+        emit MessageProcessedEvent(message);
     }
 
+    // Is this the regular way to upgrade?
     // TODO: going to need to test this method of upgrading
     // note: upgrading this way will not enable initialization using "upgradeToAndCall". explore more
     function _upgrade(address upgradeImplementation) internal returns (bool, bytes memory) {
         return address(bridge).call(
             abi.encodeWithSignature("upgradeTo(address)", upgradeImplementation)
         );
+        // emit event
     }
 
     function _transferBridgeOwnership(address newOwner) internal {
         bridge.transferOwnership(newOwner);
+        // emit event
     }
 
+    // Should we check the original value, for example, if the member is already blocklisted, 
+    // then a subsequent blocklist will fail. It feels more conceptually right
```

Comment:
We could! does it make sense that supplying a duplicate blocklisted address should revert the transaction?